py/selenium/webdriver/phantomjs/service.py

```diff
@@ -75,13 +75,15 @@ def start(self):
                                             stdout=self._log, stderr=self._log)
 
         except Exception as e:
-            raise WebDriverException("Unable to start phantomjs with ghostdriver.", e)
+            raise WebDriverException(
```

Comment:
Thanks for your feedback.
`stacktrace` does not appear to be used in other places where the exception is raised?
(neither is `msg` being used as kwargs)
I will push a fixup-commit for this though.